src/lib/sessions.ts

```diff
@@ -76,3 +76,15 @@ export function getSessionTitle(session: Session) {
 	const hasKnowledge = session.messages[0]?.knowledge;
 	return hasKnowledge ? session.messages[1]?.content : session.messages[0]?.content;
 }
+
+export function getLastUsedModels() {
+	const currentSessions = get(sessionsStore) || [];
+	const lastUsedModels: string[] = [];
+
+	for (const session of currentSessions) {
+		if (!lastUsedModels.includes(session.model)) lastUsedModels.push(session.model);
+		if (lastUsedModels.length >= 5) break;
+	}
+
+	return lastUsedModels;
+}
```

Comment:
We are only using this function in `FieldSelectModel` so let's keep this logic there, maybe rename it to `handleRecentlyUsedModels` so it's consistent with `handleChange()` and the label on the screen.

If we find we need to use this in other places later we might end up moving it back to `sessions.ts`.